packages/sdk/src/utils.ts

```diff
@@ -575,20 +574,36 @@ export async function fetchPrice(identifier: string, chainId = 1, apiKey?: strin
 
 	const headers = apiKey ? { "x-cg-pro-api-key": apiKey as string } : undefined
 
-	const response = await fetch(url, { headers })
+	const maxRetries = 3
```

Comment:
Let's just wrqp the fetch calls in this
https://github.com/polytope-labs/hyperbridge-sdk/blob/bd0b0b81d410d37a0ec21fbff1417e523229734d/packages/sdk/src/utils.ts#L304